.bazelrc

```diff
@@ -120,6 +120,14 @@ test:ci --nocache_test_results
 test:ci --spawn_strategy=local
 test:ci --test_output=errors
 test:ci --test_verbose_timeout_warnings
+test:ci-debug -c dbg
+test:ci-debug --javacopt="-g"
+test:ci-debug --copt="-g"
+test:ci-debug --flaky_test_attempts=3
```

Comment:
Are these necessary? Isn't it possible to specify 2 configs? (e.g., config=ci --config=ci-debug)?

Reply:
cc @simon-mo